src/udev/udevadm-control.c

```diff
@@ -40,6 +40,7 @@ static int help(void) {
                "  -R --reload              Reload rules and databases\n"
                "  -p --property=KEY=VALUE  Set a global property for all events\n"
                "  -m --children-max=N      Maximum number of children\n"
+               "     --ping                Send a ping message\n"
```

Comment:
Maybe describe this as "Wait for udev to respond to a ping message"

Reply:
Thanks.